denops/@denops-private/cli_main.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  readableStreamFromWorker,
+  writableStreamFromWorker,
+} from "jsr:@lambdalisue/workerio@4.0.0";
+import { parseArgs } from "jsr:@std/cli/parse-args";
```

Comment:
@lambdalisue Is it intentional that `jsr:@std/cli` has no version specification?

Reply:
No. Unintentional.